mountpoint-s3-crt-sys/Cargo.toml

```diff
@@ -28,13 +28,20 @@ exclude = [
     "crt/aws-lc/crypto/fipsmodule/bn/test/*",
     "crt/aws-lc/crypto/fipsmodule/policydocs/*",
     "crt/aws-lc/crypto/fipsmodule/sha/testvectors/*",
+    "crt/aws-lc/crypto/fipsmodule/ml_kem/kat/*",
```

Comment:
Can we scope this and similar entries down to just `*.txt` and also comment why we think they are safe to remove?

i.e. we identified all the text files in this directory are used for tests.